engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -92,20 +92,21 @@ object WorkflowManagerActor {
  * WorkflowOutputs: Returns a `Future[Option[binding.WorkflowOutputs]]` aka `Future[Option[Map[String, WdlValue]]`
  *
  */
-class WorkflowManagerActor(config: Config)
+class WorkflowManagerActor(isServerMode: Boolean, config: Config = WorkflowManagerActor.defaultConfig)
   extends LoggingFSM[WorkflowManagerState, WorkflowManagerData] with CromwellActor {
 
-  def this() = this(WorkflowManagerActor.defaultConfig)
   implicit val actorSystem = context.system
 
   private val logger = Logging(context.system, this)
   private val tag = "WorkflowManagerActor"
 
   private val donePromise = Promise[Unit]()
 
+  val isRestartable = config.getConfig("backend").getBooleanOr("workflow-restart", default = true)
```

Comment:
It would blow up for the missing backend stanza but can we actually skip that stanza in the configs?